.gitignore

```diff
@@ -10,7 +10,9 @@ docs/schema.json
 node_modules
 .idea
 .env
-images
+image/*
+videos/*
```

Comment:
Ensure that:

1. any subdirectories in this directory also gets ignored
2. it only affects the directory in root

Reply:
done